src/templates/skis.tsx

```diff
@@ -117,10 +114,9 @@ const SkiFinder = ({ document }: TemplateRenderProps) => {
   >([]);
   const [reviews, setReviews] = useState<ReviewProfile[]>([]);
 
-  console.log(document);
+  const sizingGuidePdfUrl = c_sizingGuide?.[0].c_sizingGuidePDF?.url;
```

Comment:
this syntax is confusing me: `?.[0].c_sizingGuidPDF` why the dot in front of the square brack?

Reply:
That means that if `c_sizingGuide` is not null, grab the first object in the list. This prevents a null exception.

Reply:
Like does the index exist